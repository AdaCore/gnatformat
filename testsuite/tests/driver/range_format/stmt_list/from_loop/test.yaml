description: StmtList formatting with partial selection.
driver: gnatformat
args:
  [
    "main.adb",
    "--range-format",
    "-SL 6",
    "-EL 12",
    "--pipe",
  ]
