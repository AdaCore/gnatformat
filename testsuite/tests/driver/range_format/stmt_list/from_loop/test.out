main.adb(<StmtList main.adb:7:7-11:12>) - 7:1-11:12
^
      A := 1;
      -- Comment;
      B := 2;$

